lib/jobs.js

```diff
@@ -1,6 +1,14 @@
 'use strict';
 
+process.on('unhandledRejection', (reason, p) => {
```

Comment:
Oops, yeah, that's just for debugging. We'll remove it before submitting the final PR